Doxyfile

```diff
@@ -638,7 +639,7 @@ SORT_MEMBER_DOCS       = YES
 # this will also influence the order of the classes in the class list.
 # The default value is: NO.
 
-SORT_BRIEF_DOCS        = NO
+SORT_BRIEF_DOCS        = YES
```

Comment:
Suggest removing "below" from [mongocxx::v_noabi::mongocxx::model::write::type()](https://github.com/mongodb/mongo-cxx-driver/blob/edc22ae9072106cce7b1b1fd2cd1ae275c2a5d98/src/mongocxx/include/mongocxx/v_noabi/mongocxx/model/write.hpp#L90-L91) since documentation sorts below other methods.

Reply:
Removed the order-dependent language + `get_*()` precondition documentation (it is already documented by the `get_*()` functions).